src/frontend/js/components/Register.js

```diff
@@ -46,6 +46,7 @@ class Register extends React.Component {
   }
 
   showPasswordsNotMatch () {
+    setTimeout(() => { this.setState({passwordsNotMatch: false}) }, 2000)
```

Comment:
This timeout should get cancelled if a user types another that doesn't match within 2 seconds